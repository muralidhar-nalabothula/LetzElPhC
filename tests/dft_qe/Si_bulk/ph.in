phonons on a grid 
 &inputph 
  nq1 = 4, 
  nq2 = 4, 
  nq3 = 4, 
  fildyn = 'si.dyn', 
  tr2_ph = 1e-14, 
  prefix = 'si', 
  trans = .true., 
  electron_phonon = 'dvscf', 
  fildvscf = 'dvscf', 
  ldisp = .true., 
  reduce_io = .true.
/ 
