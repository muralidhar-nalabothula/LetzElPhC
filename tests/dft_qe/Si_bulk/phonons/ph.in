phonons on a grid 
 &inputph 
  nq1 = 4, 
  nq2 = 4, 
  nq3 = 4, 
  fildyn = 'si.dyn', 
  tr2_ph = 1e-16, 
  prefix = 'si', 
  trans = .true., 
  electron_phonon = 'dvscf', 
  fildvscf = 'dvscf', 
  ldisp = .true., 
  !lraman = .true.
  alpha_mix(1) = 0.5, 
  reduce_io = .true.
/ 
