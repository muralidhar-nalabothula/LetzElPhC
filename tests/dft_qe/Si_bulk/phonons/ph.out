
     Program PHONON v.7.3 starts on  6Mar2024 at  6: 9:22 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     4 processors

     MPI processes distributed on     1 nodes
     0 MiB available memory on the printing compute node when the environment starts

     Waiting for input...
     Reading input from standard input

     Reading xml data from directory:

     ./si.save/

     R & G space division:  proc/nbgrp/npool/nimage =       4
     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used


     IMPORTANT: XC functional enforced from input :
     Exchange-correlation= PBE
                           (   1   4   3   4   0   0   0)
     Any further DFT definition will be discarded
     Please, verify this is what you really want


     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min         145     145     46                 2351     2351     420
     Max         146     146     47                 2354     2354     423
     Sum         583     583    187                 9409     9409    1687

     Using Slab Decomposition

     Reading collected, re-writing distributed wavefunctions in ./


     Dynamical matrices for ( 4, 4, 4)  uniform grid of q-points
     (   8 q-points):
       N         xq(1)         xq(2)         xq(3) 
       1   0.000000000   0.000000000   0.000000000
       2  -0.250000000   0.250000000  -0.250000000
       3   0.500000000  -0.500000000   0.500000000
       4   0.000000000   0.500000000   0.000000000
       5   0.750000000  -0.250000000   0.750000000
       6   0.500000000   0.000000000   0.500000000
       7   0.000000000  -1.000000000   0.000000000
       8  -0.500000000  -1.000000000   0.000000000

     Number and degeneracy of irreps per q-point
       N         xq(1)         xq(2)         xq(3)   N irreps
       1   0.000000000   0.000000000   0.000000000       2
        3   3
       2  -0.250000000   0.250000000  -0.250000000       4
        1   1   2   2
       3   0.500000000  -0.500000000   0.500000000       4
        1   1   2   2
       4   0.000000000   0.500000000   0.000000000       4
        1   1   2   2
       5   0.750000000  -0.250000000   0.750000000       6
     No degeneracy
       6   0.500000000   0.000000000   0.500000000       6
     No degeneracy
       7   0.000000000  -1.000000000   0.000000000       3
        2   2   2
       8  -0.500000000  -1.000000000   0.000000000       3
        2   2   2

     Saving dvscf to file. Distribute only q points, not irreducible representations.

     Calculation of q =    0.0000000   0.0000000   0.0000000

     phonons on a grid                                                          

     bravais-lattice index     =            0
     lattice parameter (alat)  =      10.3346  a.u.
     unit-cell volume          =     275.9441 (a.u.)^3
     number of atoms/cell      =            2
     number of atomic types    =            1
     kinetic-energy cut-off    =      40.0000  Ry
     charge density cut-off    =     160.0000  Ry
     convergence threshold     =      1.0E-16
     beta                      =       0.5000
     number of iterations used =            4
     Exchange-correlation= PBE
                           (   1   4   3   4   0   0   0)


     celldm(1)=   10.33460  celldm(2)=    0.00000  celldm(3)=    0.00000
     celldm(4)=    0.00000  celldm(5)=    0.00000  celldm(6)=    0.00000

     crystal axes: (cart. coord. in units of alat)
               a(1) = ( -0.5000  0.0000  0.5000 )  
               a(2) = (  0.0000  0.5000  0.5000 )  
               a(3) = ( -0.5000  0.5000  0.0000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = ( -1.0000 -1.0000  1.0000 )  
               b(2) = (  1.0000  1.0000  1.0000 )  
               b(3) = ( -1.0000  1.0000 -1.0000 )  


     Atoms inside the unit cell: 

     Cartesian axes

     site n.  atom      mass           positions (alat units)
        1  Si     28.0850   tau(    1) = (    0.00000    0.00000    0.00000  )
        2  Si     28.0850   tau(    2) = (    0.25000   -0.25000   -0.25000  )

     Computing dynamical matrix for 
                    q = (   0.0000000   0.0000000   0.0000000 )

     49 Sym.Ops. (with q -> -q+G )


     G cutoff =  432.8602  (   2351 G-vectors)     FFT grid: ( 32, 32, 32)
     number of k points=    29

     PseudoPot. # 1 for Si read from file:
     ../../pseudos_dojo/Si.upf
     MD5 check sum: 02fab3f35e82123ef5bf1cb05d5b1a5e
     Pseudo is Norm-conserving + core correction, Zval =  4.0
     Generated using ONCVPSP code by D. R. Hamann
     Using radial grid of 1510 points,  6 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
                l(5) =   2
                l(6) =   2

     Mode symmetry, O_h (m-3m)  point group:


     Electric field:
     Dielectric constant
     Born effective charges in two ways 


     Atomic displacements:
     There are    2 irreducible representations

     Representation     1      3 modes -  To be done

     Representation     2      3 modes -  To be done



     Alpha used in Ewald sum =   1.7000
     PHONON       :      0.07s CPU      0.10s WALL


     Electric Fields Calculation
